### ISSUE TITLE
Inconsistent code and doc for `atLeast`/`atMost`

### ISSUE THREAD

**Issue by igormoreno** [diagrams/diagrams-core#117]
Hi,

I think there is a problem with the documentation and the naming of the functions `atLeast` and `atMost`.

code:
https://github.com/diagrams/diagrams-core/blob/c37340d35da573b647bc0c5e544815002dd265f8/src/Diagrams/Core/Measure.hs#L81-L87

manual:
https://diagrams.github.io/doc/manual.html#measurement-expressions

Let's take `atLeast`. We have:
- The function is implemented with `max`.
- The function documentation says "smaller".
- The manual says the "maximum".
- The meaning of "at least" should be "smaller" IMHO. For example, if you have two measurements 5 and 10 then you have at least 5, not at least 10.

**Comment by igormoreno** [comment]
Oh... I see. I had a different understanding of "at least" :-) ok got it. So there is a "preference" for the first argument. ok. Should I change the comment in a pull request? I didn't change it before because I thought it was going to be a bigger change.

**Comment by byorgey** [comment]
There's actually no bias towards any particular argument, but I find it is easiest to think of it that way.  And sure, a PR to change the Haddock comments would be great!

**Comment by igormoreno** [comment]
Right. The implementation is with `max`, which doesn't have a bias towards any argument. Happy to help!
